libflux/flux/src/lib.rs

```diff
@@ -1,4 +1,6 @@
-#![cfg_attr(feature = "strict", deny(warnings))]
+#![cfg_attr(feature = "strict", deny(warnings, missing_docs))]
+
+//! Serialization from Flux.
```

Comment:
Maybe

```suggestion
//! Serialization of Flux code.
```
?

Does this module do other work?

Reply:
Although maybe this is okay as is...

Reply:
I was about to comment to this effect. This module provides the public facing API for the Go runtime including formatting, parsing and stdlib-aware analysis.